austrakka/components/iam/role/funcs.py

```diff
@@ -0,0 +1,118 @@
+from austrakka.components.iam.shared_funcs import _get_default_tenant, _get_role_by_name
+from austrakka.utils.api import api_get, api_post, api_patch
+from austrakka.utils.enums.privilege_level import (
+    AUSTRAKKA_ADMIN_LEVEL,
+    FUNCTIONAL_ADMIN_LEVEL,
+    USER_LEVEL)
+
+from austrakka.utils.misc import logger_wraps
+from austrakka.utils.output import print_dict
+
+
+@logger_wraps()
+def get_roles(out_format: str):
+    """
+    Get the list of roles defined for a tenant.
+    """
+    tenant_global_id = _get_default_tenant()
+    response = api_get(
+        path=f"v2/tenant/{tenant_global_id}/role",
+    )
+
+    data = response['data'] if ('data' in response) else response
+    print_dict(
+        data,
+        out_format,
+    )
+
+
+@logger_wraps()
+def add_role(role: str, description: str, privilege_level: str, allowed_record_types: list[str]):
```

Comment:
same with this will we have like an `add_project_role` or something in the future?

Reply:
Vanity function. For future implementation.